helmfile/charts/gatekeeper/podsecuritypolicies/templates/restricted/mutation/run-as-group.yaml

```
@@ -0,0 +1,69 @@
+apiVersion: mutations.gatekeeper.sh/v1
+kind: Assign
+metadata:
+  name: mutation-run-as-group
+spec:
+  applyTo:
+    - groups: [""]
+      kinds: ["Pod"]
+      versions: ["v1"]
+  match:
+    scope: Namespaced
+    kinds:
+      - apiGroups: ["*"]
+        kinds: ["Pod"]
+    namespaces: {{- toYaml .Values.namespaces | nindent 6 }}
+    labelSelector:
+      matchExpressions:
+        {{- if .Values.mutation }}
+          {{- range $value := .Values.mutation }}
+            {{- range $labelkey, $labelvalue := $value }}
+        - key: {{ $labelkey }}
+          operator: NotIn
+          values:
+            - {{ $labelvalue | quote }}
+            {{- end }}
+          {{- end }}
+        {{- end }}
+  location: "spec.containers[name: *].securityContext.runAsGroup"
+  parameters:
+    assign:
+      value: 1000
```

```suggestion
      value: 10000
```
Seems to be a recommendation from certain sources since 1000 could easily exist on the host. Don't know if that would break something thou.

---

Sure, i can try it

---

I had issue with the exporters i updated `your UID isn't in the passwd file.`
But i can just specify runAsUser: 1000 for it.

---

I think I have to back-pedal on this due to upstream charts, many assume 1000, won't set it properly and won't allows us to modify it. :no_mouth: